.goreleaser.yml

```diff
@@ -61,12 +61,27 @@ builds:
       -X=github.com/bomctl/bomctl/cmd.BuildDate={{.Date}}
 
 signs:
-  - id: cosign
-    signature: $artifact.sig
+  - id: checksum-keyless-signing
+    signature: "${artifact}-keyless.sig"
+    certificate: "${artifact}-keyless.pem"
     cmd: cosign
-    args: [sign-blob, --output-signature=$artifact.sig, --key=env://COSIGN_PRIVATE_KEY, $artifact]
+    args: [sign-blob, --output-signature, "${artifact}-keyless.sig", --output-certificate, "${artifact}-keyless.pem", $artifact]
     artifacts: checksum
 
+  - id: binary-keyless-signing
+    signature: "${artifact}-keyless.sig"
+    certificate: "${artifact}-keyless.pem"
+    cmd: cosign
+    args: [sign-blob, --output-signature, "${artifact}-keyless.sig", --output-certificate, "${artifact}-keyless.pem", $artifact]
```

Comment:
Will this work?

```suggestion
    args: [sign-blob, --output-signature=$signature, --output-certificate=$certificate, $artifact]
```

Reply:
Maybe theirs was originally written before goreleaser added support for that variable?